docs/getting-started.md

```diff
@@ -211,10 +254,13 @@ The cause of the _overscanning_ can be found in the [repository field of acorn's
 },
 ```
 
-One can not fix the sources of already released package artifacts so to resolve the missing _directory_ in the metadata of `NPM::acorn:7.1.1` you can use a [VCS path curation][ort-curations]:
+Although it is not possible to fix the sources of previously released package artifacts, you can resolve the missing _directory_ in the metadata
```

Comment:
What does "resolving a the missing directory" actually mean?

Reply:
Changed 'resolve' to 'add'